docs/source/configuration.rst

```diff
@@ -75,6 +75,39 @@ Queue Rules
 
 See :ref:`queue rules` for the complete list and description of options.
 
+.. _command restrictions:
+
+Command Restrictions
+~~~~~~~~~~~~~~~~~~~~
+
+- The value type of the ``command_restrictions`` is dictionary.
+
+- Each entry in ``command_restrictions`` must be an :ref:`Actions` name and a dictionary.
+
+Each dictionary must have the following keys:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 1 1 2
+
+   * - Key Name
+     - Value Type
+     - Value Description
+   * - ``conditions``
+     - list of :ref:`Conditions`
+     - A list of :ref:`Conditions` string that must match against the pull
+       request for the command to be allowed.
+
+For example, to limit backport command for pull request targeting the main branch:
```

Comment:
This is unclear. To me, the `conditions` match on the source pull request. They cannot match on a destination pull request since it does not exist yet 🤔 
Reading the code, this is what it actually does IIRC, so I think this explanation is just unclear.

```suggestion
For example, to limit backport commands for pull requests coming from the main branch:
```